lib/di/global/get_it_initializer.dart

```diff
@@ -15,8 +16,9 @@ class GetItInitializer {
   GetItInitializer._internal();
 
   void setUp() {
-    NetworkDI().bind();
```

Comment:
Wwhy neeed to remove it?

Reply:
I just change the order because Hive db should be in first

Reply:
No. Be careful in the order of DI. The best way is don't touch